packages/bierzo-wallet/package.json

```diff
@@ -28,7 +30,9 @@
     "format": "prettier --write --loglevel warn './src/**/*.{ts,tsx,json,md,css}'",
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "CI=true react-scripts test '\\.dom\\.(spec|test)\\.[jt]s[x]?$' --env=jsdom"
+    "test-dom": "CI=true react-scripts test '\\.dom\\.(spec|test)\\.[jt]s[x]?$' --env=jsdom",
+    "test-e2e": "CI=true concurrently -s \"first\" -k \"serve -s build -l 3000\" \"react-scripts test '\\.e2e\\.(spec|test)\\.[jt]s[x]?$' --env=jsdom\"",
```

Comment:
What is `concurrently -s \"first\" -k \"serve -s build -l 3000\"`?

Reply:
full command is `concurrently -s "first" -k "serve -s build -l 3000" "react-scripts test '\\.e2e\\.(spec|test)\\.[jt]s[x]?$' --env=jsdom"
`concurrently` - run commands in parallel 
`-s "first" -k` - kill all threads after first finished thread and return command execution status code from first finished thread.
`serve -s build -l 3000` - serves web static content from `build` directory on port 3000
`react-scripts test '\\.e2e\\.(spec|test)\\.[jt]s[x]?$' --env=jsdom` - I think you know what it is for.